src/modules/partners/partner_test.ts

```diff
@@ -208,12 +202,13 @@ describe('test partners using cursor', () => {
             data: expect.arrayContaining([
               expect.objectContaining({
                 id: expect.any(String),
-                name: 'TokoPedia',
+                name: expect.any(String),
               }),
             ]),
             meta: expect.objectContaining({
-              per_page: 1,
-              next_page: minuteBeforeTimestamp(1, timestamp).toISOString(),
+              per_page: expect.any(Number),
+              total: expect.any(Number),
+              next_page: expect.any(String),
```

Comment:
I prefer these fields (at least `name` and `next_page`) to be hardcoded (since it's a test, it should test a specific thing). With this `expect.any`, if someone changes the logic and the result is `TokoCrypto` instead of `TokoPedia` (or maybe a wrong date), no one will know and no one will aware of it.

Reply:
that's because the existing data has not been deleted so it's not certain the data we want to come out, if you want a suitable search can add a query name to match what we defined, but PR now improve date to moment js, :smile: